phalcon/flash/session.zep

```diff
@@ -67,7 +67,11 @@ class Session extends FlashBase implements FlashInterface, InjectionAwareInterfa
 		let session = <SessionInterface> dependencyInjector->getShared("session");
 		let messages = session->get("_flashMessages");
 
-		if typeof type == "string" && isset(messages[type]) {
+		if typeof type == "string" {
+			if !isset(messages[type])) {
+				return [];
+			}
+			
 			if !fetch returnMessages, messages[type] {
```

Comment:
@Jurigag I'm not familiar to code Zephir. I looked up behavior of `fetch` operator and you are right.

Following block possibly better:
```zephir
if typeof type == "string" {
    if fetch returnMessages, messages[type] {
        if remove === true {
            unset(messages[type]);
            session->set("_flashMessages", messages);
        }
        return returnMessages;
    }
    
    return [];
}
```

I'm going to commit this.